####################################################
# Author:                 J. Gong
# Date:                      2021-02-04
# Description:        Configuration for 'lidar_surface_detector'
####################################################

# Lidar mountage
MOUNT_HEIGHT: 0.7
MOUNT_ANGLE: 0.0
TARGET_WIDTH: 1.0
TARGET_WIDTH_RESOLUTION: 0.1
TARGET_LOOKAHEAD: 5.0

# Pattern
PATTERN_WINDOW: 2.5
PATTERN_INTERVAL: 0.1
SCAN_FREQUENCY: 10
AMPLITUDE_THRESHOLD: 0.015
POSITIVE_FIRST: False